commands/http/handler.go

```diff
@@ -56,10 +57,8 @@ const (
 )
 
 var localhostOrigins = []string{
-	"http://127.0.0.1",
-	"https://127.0.0.1",
-	"http://localhost",
-	"https://localhost",
+	"127.0.0.1:5001",
+	"localhost:5001",
 }
```

Comment:
We should probably get the API port from the config rather than hardcoding port 5001.